src/Orleans/Logging/LoggerWrapper.cs

```diff
@@ -0,0 +1,179 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+
+namespace Orleans.Runtime
+{
+    //TODO: Mark it as [Obsolete] after all runtime has migrated
+    internal class LoggerWrapper : Logger
+    {
+        public override Severity SeverityLevel => this.maxSeverityLevel;
+
+        public override string Name => this.name;
+        private string name;
+        private readonly Severity maxSeverityLevel;
+        private readonly ILogger logger;
+        private readonly IPEndPoint ipEndpoint;
+        private readonly ILoggerFactory loggerFactory;
+        public LoggerWrapper(string name, ILoggerFactory loggerFactory, IPEndPoint ipEndpoint)
```

Comment:
IS IPEndPoint ever passed into this constructor? It can probably be removed, right?

Reply:
Yes, it will be passed in if there's one in DI. In current PR client side doesn't have ipEndPoint concept, that's why it is null. But ipEndPoint is needed there to achieve legacy logging message style

Reply:
We synced in person. We decided that LoggerWrapper should not try to pre-format everything as the original LogConsumer sinks. It will just forward to ILogger and drop the IPEndPoint and manage thread id, and all that contextual info.
If the end user is also using the other back compat feature (added in the other PR) which keeps using `ILogConsumer`, then that contextual information will be added at that point. But if the end user configures the new `ILogProvider` sinks from MEL instead, then it's up to those sinks to enhance with whatever contextual information it wants.